.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,20 @@
+name: Gradle Build
```

Comment:
Could you please rebase this branch on the latest master to exclude these changes?